scripts/generate_proto_stubs.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
Could you add a comment explaining the purpose of this file more directly?

Reply:
yep! I elaborate on the comment - describing that the script autogenerates the _pb2.pyi files

Reply:
Thanks, sounds good!